css/style.css

```diff
@@ -1 +1,1461 @@
-body {}
\ No newline at end of file
+:root {
+  --btn-primary-color: rgba(33, 231, 134, 1);
+  --secondary-color: rgba(20, 27, 34, 1);
+  --article-bg-color: rgba(255, 255, 255, 0.05);
+  --main-background-color: rgba(13, 13, 17, 1);
+  --text-color: rgba(194, 195, 197, 1);
+  --inactive-color: rgba(136, 139, 142, 1);
+  --text-font: "Poppins", "Helvetica Neue", Arial, sans-serif;
+  --btn-header-font: "Bakbak One", "Arial Black", Arial, sans-serif;
+}
+
+/********************** General css **********************************/
+
+* {
+  padding: 0;
+  margin: 0;
+}
+
+body {
+  background-color: rgba(4, 11, 17, 1);
+  font-family: "Bakbak One", "Arial Black", Arial, sans-serif;
+  font-weight: 400;
+  font-style: normal;
+  font-size: 18px;
+  line-height: 28px;
```

Comment:
it is better if the `line-height` is specified in dimensionless units - in this case `line-height: 1.56;` (28px / 18px)

Reply:
Thanks, I agree and see the reason why it's better. But is there a tool that calculates it? because i have only dimension size in figma and it's a crazy amount of font-size/line-height combinations in this design...

Reply:
I use calculator for it - I don't know more convenient tool)

Reply:
#fixed